Source/WebCore/Modules/model-element/DDModelPlayer.h

```diff
@@ -95,6 +100,7 @@ class WEBCORE_EXPORT DDModelPlayer final : public ModelPlayer {
     WeakRef<Page> m_page;
     mutable RefPtr<ModelDisplayBufferDisplayDelegate> m_contentsDisplayDelegate;
     uint32_t m_currentTexture { 0 };
+    std::optional<bool> m_paused;
```

Comment:
I try to steer people away from std::optional<bool>, as I find it hard to reason about and usually an enum is better suited.